sqlglot/parser.py

```diff
@@ -5912,6 +5912,8 @@ def _parse_auto_increment(
     ) -> exp.GeneratedAsIdentityColumnConstraint | exp.AutoIncrementColumnConstraint:
         start = None
         increment = None
+        show_order = None
+        order = None
```

Comment:
This doesn't look right, we should only introduce `order` and then treat `None` as "does not exist" / "don't show".